main.nf

```diff
@@ -0,0 +1,684 @@
+#!/usr/bin/env nextflow
+/*
+========================================================================================
+                         nf-core/rnafusion
+========================================================================================
+ nf-core/rnafusion Analysis Pipeline.
+ #### Homepage / Documentation
+ https://github.com/nf-core/rnafusion
+----------------------------------------------------------------------------------------
+*/
+
+def helpMessage() {
+    log.info"""
+    =========================================
+     nf-core/rnafusion v${workflow.manifest.version}
+    =========================================
+    Usage:
+
+    The typical command for running the pipeline is as follows:
+
+    nextflow run nf-core/rnafusion --reads '*_R{1,2}.fastq.gz' -profile standard,docker
+
+    Mandatory arguments:
+      --reads                       Path to input data (must be surrounded with quotes)
+      --genome                      Name of iGenomes reference
+      -profile                      Configuration profile to use. Can use multiple (comma separated)
```

Comment:
insn't standard the defaul profile if you don't specify anything?
Then is it really mandatory?

Reply:
We don't have a standard profile in `.nextflow.config`

Reply:
yes, `base.config`